crates/cli-tools/src/changelog.rs

```diff
@@ -266,19 +271,56 @@ struct Release {
     contents: BTreeMap<Severity, Vec<String>>,
 }
 
-impl Release {
-    fn push_content(&mut self, severity: Severity, content: &str) -> Result<()> {
-        self.contents.entry(severity).or_default().push(content.to_string());
-        Ok(())
-    }
-}
-
 impl From<Version> for Release {
     fn from(version: Version) -> Self {
         Release { version, contents: BTreeMap::new() }
     }
 }
 
+impl Release {
+    fn get_max_severity(&self) -> Severity {
+        for severity in [Severity::Major, Severity::Minor, Severity::Patch] {
+            if self.contents.contains_key(&severity) {
+                return severity;
+            }
+        }
+
+        Severity::Patch
```

Comment:
1. Keys are sorted in a BTreeMap.
2. We have an invariant that contents is not empty.
```suggestion
        self.contents.keys().next().unwrap()
```

Reply:
Ahh nice!